device/dell/x86_64-dellemc_s5248f_c3538-r0/DellEMC-S5248f-P-25G/gearbox_config.json

```diff
@@ -0,0 +1,38 @@
+{
+  "phys": [
+    {
+      "phy_id": 1,
+      "name": "sesto-1",
```

Comment:
i think we need to add a type here, in this case based on the type, we can load different phy driver.

Reply:
Definitely can be considered